internal/modelsutil/vulnerabilities.go

```diff
@@ -0,0 +1,20 @@
+package modelsutil
```

Comment:
How about naming this packgage `vulns` instead? This is similar to e.g. `bytes`, `strings` in the Go std library. 

This way we also avoid a lot of stutter in the funciton names. 

e.g. vulns.Includes() vs modeslutil.VulnsIncludes()